Applications/MainApplication/MainApplication.cpp

```diff
@@ -278,11 +280,12 @@ namespace Ra
 
         // ----------
         // 2. Kickoff rendering
+        m_viewer->m_dt = dt;
+//        m_viewer->update();
         m_viewer->startRendering( dt );
 
-
+/*

```

Comment:
I don't like the idea of running  the tasks in the UI thread by overloading QWidget::update(), because then the whole UI is unresponsive when you run heavy tasks.